### ISSUE TITLE
Buoy Dashboard Widget should become an "At A Glance" widget item or several

### ISSUE THREAD

**Issue by fabacab** [betterangels/buoy#131]
Some of the information in the "Buoy Dashboard Widget" might be better suited to the "At A Glance" widget. For instance, the "You have responders on this buoy" is a prime candidate.

See https://developer.wordpress.org/?s=glance&post_type[]=wp-parser-hook for some relevant hook reference, and the [`dashboard_glance_items` filter hook](https://developer.wordpress.org/reference/hooks/dashboard_glance_items/) specifically.

**Comment by hurtstotouchfire** [comment]
Still learning WP. This widget shows up on all admin dashboards? (I'm looking at https://codex.wordpress.org/File:dashboard.png as an example.)

This seems like it might be another UI way to help with the "Where is buoy" question?

**Comment by fabacab** [comment]
> This widget shows up on all admin dashboards?

Yes, *as long as* the WordPress site is in fact allowing all users to view this Dashboard screen. (Some aggressively modified WordPress sites don't allow this, but I think we can consider those edge cases.) Those rectangular blocks in the image you linked can be rearranged and collapsed/expanded, and are what WordPress calls "Dashboard Widgets." This is distinct from what's termed an "'At a Glance' item," or so I learned when I created this issue, which is a specific line item inside the Dashboard Widget specifically *named* `At a Glance`, and usually shows up at the top-left of a user's Dashboard screen (as shown in the image you linked).

> This seems like it might be another UI way to help with the "Where is buoy" question?

That's my hope.
